src/dataset-builder/CohortEditor.ts

```diff
@@ -58,24 +59,23 @@ export const CriteriaView = (props: CriteriaViewProps) => {
   const [criteriaCount, setCriteriaCount] = useLoadedData<SnapshotBuilderCountResponse>();
 
   const updateCriteriaCount = useRef(
-    _.debounce(250, (snapshotId, criteria) => {
+    debounceAsync((snapshotId: string, criteria: AnyCriteria) =>
       setCriteriaCount(
-        withErrorReporting('Error fetching count for criteria')(
-          async () =>
-            await DataRepo()
-              .snapshot(snapshotId)
-              .getSnapshotBuilderCount(
-                createSnapshotBuilderCountRequest([
-                  {
-                    // Create a "cohort" to get the count of participants for these criteria on its own.
-                    criteriaGroups: [{ id: 0, criteria: [criteria], meetAll: true, mustMeet: true }],
-                    name: '',
-                  },
-                ])
-              )
+        withErrorReporting('Error getting criteria group count')(async () =>
+          DataRepo()
```

Comment:
With no longer having `await` here, you can get rid of the `async` above!

Reply:
Oh, I don't think I meant to remove this async/await. I instead added the await back. 
Thanks for catching that!